crates/notary/server/tee/notary-server-sgx.md

```diff
@@ -0,0 +1,68 @@
+# TLSNotary Notary Server for Intel SGX
```

Comment:
should we also remove the original README.md (also in this tee folder) to avoid confusion?

Reply:
No. I'll improve this one later in https://github.com/tlsnotary/tlsn/pull/695
This PR focusses on CI (and the resulting zip file) only